data/lib/core/player.lua

```diff
@@ -100,3 +100,33 @@ function Player.addManaSpent(...)
 	APPLY_SKILL_MULTIPLIER = true
 	return ret
 end
+
+function Player.transferMoneyTo(self, target, amount)
+	local balance = self:getBankBalance()
+	if amount > balance then
+		return false
+	end
+
+	local targetPlayer = Player(target)
+	if targetPlayer then
+		targetPlayer:setBankBalance(targetPlayer:getBankBalance() + amount)
+	else
+		if not playerExists(target) then
+			return false
+		end
+		db.query("UPDATE `players` SET `balance` = `balance` + '" .. amount .. "' WHERE `name` = " .. db.escapeString(target))
```

Comment:
The single quotes around `amount` are not necessary but not wrong either.
This function doesn't prevent any SQL injection. I know there is `isValidMoney` use in the NPC though.
A comment should be added to warn the user to always pass the number through that function first.